pom.xml

```diff
@@ -1882,6 +1883,32 @@
       </plugins>
     </pluginManagement>
     <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>flatten-maven-plugin</artifactId>
+        <!--<version>1.3.0</version>-->
```

Comment:
This is because the version has been declared in the parent apache pom?

Reply:
Use 'latest'. Will set it to 1.3.0 explicitly next go around.

Reply:
I have mixed feelings about manually managing plugin versions but agree it is best on balance. In our own internal builds we need to manage things up sometimes and the version property definitions for plugin versions are helpful for that. Other downstreamers will have a similar experience I suspect.